docs/pages/includes/install-linux.mdx

```diff
@@ -44,15 +44,15 @@ Next, use the appropriate commands for your environment to install your package.
   # Note: if using a fork of RHEL/CentOS or Amazon Linux you may need to use '$ID_LIKE'
   # and the codename your distro was forked from instead of '$ID'
   # Supported versions are listed here: https://github.com/gravitational/teleport/blob/master/build.assets/tooling/cmd/build-os-package-repos/runners.go#L133-L153
-  $ sudo dnf config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
-  $ sudo dnf install teleport
+  $ sudo yum-config-manager --add-repo $(rpm --eval "https://yum.releases.teleport.dev/$ID/$VERSION_ID/Teleport/%{_arch}/stable/v(=teleport.major_version=)/teleport.repo")
+  $ sudo yum install teleport
   #
   # Tip: Add /usr/local/bin to path used by sudo (so 'sudo tctl users add' will work as per the docs)
   # echo "Defaults    secure_path = /sbin:/bin:/usr/sbin:/usr/bin:/usr/local/bin" > /etc/sudoers.d/secure_path
   #
-  # Optional:  Using YUM on older distributions
-  #$ sudo yum-config-manager --add-repo $(rpm --eval "https://yum.releases.teleport.dev/$ID/$VERSION_ID/Teleport/%{_arch}/stable/v(=teleport.major_version=)/teleport.repo")
-  #$ sudo yum install teleport
+  # Optional:  Using DNF on newer distributions
```

Comment:
```suggestion
  # Optional:  Use DNF on newer distributions
```

Since we say `Tip: Add` instead of `Tip: Adding`, should we say `Optional: Use` instead of `Optional: Using`?